extensions/io/org.eclipse.smarthome.io.javasound/src/main/java/org/eclipse/smarthome/io/javasound/internal/JavaSoundAudioSink.java

```diff
@@ -118,4 +136,7 @@ private void runVolumeCommand(Closure closure) {
         }
     }
 
+    private boolean isMacOS() {
+        return System.getProperty("osgi.os").equals("macosx");
```

Comment:
is this the last resort?
You previously posted that osgi.os is not reliable to detect the OS (https://github.com/eclipse/smarthome/issues/1416#issue-151054512).